sdks/python/standard_external_transforms.yaml

```diff
@@ -0,0 +1,725 @@
+#
```

Comment:
Could we just have a single example transform, then check in all this (and the proceeding file) that adds all the transforms in as a subsequent PR?